paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/ACHText.kt

```diff
@@ -0,0 +1,30 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.ach
+
+import android.content.Context
+import com.stripe.android.paymentsheet.R
+
+/**
+ * Temporary hack to get mandate text to display properly until translations are fixed
+ */
+object ACHText {
```

Comment:
This is a temporary workaround

Reply:
What's the fix?

Reply:
The translations currently use %@ instead of the universal %s format. The fix would be to update the translations through lokalise, just not sure how long it will take